### PR TITLE
Feature/interactives latest

### PR DESCRIPTION
**Pull request by geoffmrb**
### What

Built the interactives upload screen with it hooked up to the new api developed by the backend team

### How to review

Log in to the app, in the menu, there will be a new tab called "interactives" click on that and you will be able to see.

### Who can review

me

### CODE REVIEW

File: src/app/utilities/api-clients/interactives-test.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
+import Interactives from "./interactives";
+
```
Comment (mwnicks) [inline src/app/utilities/api-clients/interactives-test.js:2]: Not sure I understand why this file is called test?

File: src/app/views/interactives/components/FooterComponent.jsx [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import React, { Component } from "react";
+
+export class FooterComponent extends Component {
+    render() {
```
Comment (mwnicks) [inline src/app/views/interactives/components/FooterComponent.jsx:4]: Is this a new footer? Is this being added to all Florence pages?
If not shouldn't we be using the same one as other parts of the app?

File: src/app/views/interactives/components/ReactTable.jsx [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+import { useTable, usePagination } from "react-table";
+import { store } from "../../../config/store";
+// from https://react-table.tanstack.com/docs/overview
+
+import { Link } from "react-router";
+
+const rootPath = store.getState().state.rootPath;
+
+function Table({ columns, data }) {
+    // Use the state and functions returned from useTable to build your UI
```
Comment (mwnicks) [inline src/app/views/interactives/components/ReactTable.jsx:11]: There is already a component for listing items, can it not be used here?
https://github.com/ONSdigital/florence/blob/develop/src/app/components/simple-selectable-list/SimpleSelectableList.jsx

File: src/img/logo-dark.svg [hunk 4]
```diff
@@ -0,0 +1,5 @@
+<svg width="213" height="20" viewBox="0 0 213 20" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (mwnicks) [inline src/img/logo-dark.svg:1]: Is there no logos in the code base already?

File: src/app/components/navbar/NavBar.jsx [hunk 5]
```diff
@@ -82,6 +82,17 @@ const NavBar = props => {
...
                             </Link>
                         </li>
+
+                        <li className="global-nav__item">
+                            <Link to={`${rootPath}/teams`} activeClassName="selected" className="global-nav__link">
+                                Teams
+                            </Link>
+                        </li>
+                        <li className="global-nav__item">
+                            <Link to={`${rootPath}/interactives/index`} activeClassName="selected" className="global-nav__link">
```
[the remark this answers is not in the document]
Comment (geoffmrb) [reply src/app/components/navbar/NavBar.jsx:92]: no reason, will amend

File: src/app/utilities/api-clients/interactives-test.js (hunk 1 above)
Comment (geoffmrb) [reply src/app/utilities/api-clients/interactives-test.js:2]: the test has specific configs for testing and importing the original interactive file to test it>

What should it be called?

File: src/app/views/interactives/components/FooterComponent.jsx (hunk 2 above)
Comment (geoffmrb) [reply src/app/views/interactives/components/FooterComponent.jsx:4]: in fact, there is no footer in florence yet, so its a new one and will be added on all florence pages as per figma

File: src/img/logo-dark.svg (hunk 4 above)
Comment (geoffmrb) [reply src/img/logo-dark.svg:1]: not this specific ones, this is new as per figma

**Review by mwnicks — APPROVED** [review]

File: src/app/actions/actionTypes.js [hunk 6]
```diff
@@ -0,0 +1,16 @@
+// Interactives
```
Comment (AnnaEmson) [inline src/app/actions/actionTypes.js:1]: Hi, I am wondering why you didn't use the current folders structure in the repository? the redux is stored in src/app/config. can you create new directory there for your object please?

**Review by AnnaEmson — CHANGES_REQUESTED** [review]
Hi, can you please move your redux code to the src/app/config as per current structure

**Pull request closed without merging**